depends/packages/qt.mk

```diff
@@ -70,19 +71,35 @@ $(package)_config_opts += -system-zlib
 $(package)_config_opts += -static
 $(package)_config_opts += -silent
 $(package)_config_opts += -v
+$(package)_config_opts += -no-feature-bearermanagement
+$(package)_config_opts += -no-feature-colordialog
+$(package)_config_opts += -no-feature-commandlineparser
+$(package)_config_opts += -no-feature-concurrent
 $(package)_config_opts += -no-feature-dial
+$(package)_config_opts += -no-feature-filesystemwatcher
+$(package)_config_opts += -no-feature-fontcombobox
 $(package)_config_opts += -no-feature-ftp
+$(package)_config_opts += -no-feature-image_heuristic_mask
+$(package)_config_opts += -no-feature-keysequenceedit
 $(package)_config_opts += -no-feature-lcdnumber
 $(package)_config_opts += -no-feature-pdf
-$(package)_config_opts += -no-feature-printer
 $(package)_config_opts += -no-feature-printdialog
-$(package)_config_opts += -no-feature-concurrent
+$(package)_config_opts += -no-feature-printer
+$(package)_config_opts += -no-feature-printpreviewdialog
+$(package)_config_opts += -no-feature-printpreviewwidget
+$(package)_config_opts += -no-feature-regularexpression
+$(package)_config_opts += -no-feature-sessionmanager
```

Comment:
As mentioned on IRC, I remember originally making this platform-specific for good reason. Maybe it's no longer needed, but let's make sure to understand what's changed.

Reply:
I've done some basic testing on macOS 10.14 and Windows 10 (built using WSL) and couldn't find obvious runtime issues. Although it might be good to get a Windows user to verify this: @sipsorcery, @NicolasDorier ?

I can't see us using or including [`QSessionManager`](https://doc.qt.io/qt-5/qsessionmanager.html) anywhere, nor making any calls to [`QGuiApplication::commitDataRequest`](https://doc.qt.io/qt-5/qguiapplication.html#commitDataRequest) or  [`QGuiApplication::saveStateRequest`](https://doc.qt.io/qt-5/qguiapplication.html#saveStateRequest) which are Qts two session management signals. My other thought was that it might be being used by `QSettings`, however config settings in the GUI (i.e coin control) seem to be saved and loaded correctly on the next start.

Looking at our qt package, `-no-sm` was originally added as a Linux only config option at the [introduction of depends](https://github.com/bitcoin/bitcoin/pull/4592/files#diff-5e50ddee0f3ab9f50378a3577075f10fR39). It then became `-no-feature-sessionmanager` during the [Qt 5.9.4 upgrade](https://github.com/bitcoin/bitcoin/pull/12971).